internal/polkavm/host_call/refine_functions_test.go

```diff
@@ -104,3 +105,66 @@ func TestHistoricalLookup(t *testing.T) {
 	expectedGasRemaining := polkavm.Gas(initialGas) - host_call.HistoricalLookupCost - polkavm.GasCosts[polkavm.Ecalli] - polkavm.GasCosts[polkavm.JumpIndirect]
 	assert.Equal(t, expectedGasRemaining, gasRemaining)
 }
+
+func TestImport(t *testing.T) {
```

Comment:
For now there is one test so this is good but when there is going to be other refine host calls we might want to write tests like in accumulate, where we have a bunch of testcases and only allocate registers/memory, this way I think is more readable and less boilerplate